Docs quality of life changes (#38)
redsigma/Unofficial-Patch-for-MetroForSteam: README.md

> # Unofficial Patch 9.1f for Metro for Steam skin

Reviewer: ```suggestion
# Unofficial Patch 9.1g for Metro for Steam skin
```
all the late changes justify for an update to version number too.

> This unofficial patch for the [Metro for Steam skin](https://metroforsteam.com) fixes bugs with current version of the skin, implements newly added Steam client features and adds optional ones for productivity and aesthetic.

Reviewer: ```suggestion
This unofficial patch for the [Metro for Steam skin](https://metroforsteam.com) fixes bugs with current version of the skin, implements newly added Steam client features and adds optional ones for customizable productivity and aesthetics.
```

Reviewer: just as I suggested.
hopefully this sounds reasonable to all.
otherwise I like the wording.

> Additional details and feedback are available on [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam group. Installation instructions can be found on the [Steam discussion thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).

Reviewer: ```suggestion
Additional details and feedback are available on [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam community group. Installation instructions can be found on the [Steam discussion thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).
```

Reviewer: where did "community" go - that was ur previous suggestion, already incorporated, lol. :)

Author: Personally don't think "community" is needed, but I've added a small correction in 1307ca1.

> Additional details and feedback are available on the [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam group. Installation instructions can be found on the [Steam discussion thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).

Reviewer: ```suggestion
Additional details and feedback are available on the [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam community group. Installation instructions can be found on the [Steam discussion thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).
```
let's keep "community".

Reviewer: btw, lose the extra spacing below the headline too, introduced by ur commits.

Author: I'd like to hear @redsigma's opinion on it

Reviewer: haven't he already decided, by making a commit introducing this into the file directly? :)
it was ur original proposal... no?

Author: I proposed to remove it in 8839a94, my original commit.

Reviewer: hmm, interesting... I actually thought it was missing originally.
anyway, I would guess he already decided since he didn't introduce it himself, but otherwise...
I'm not sure why it's so important to remove it.

in any case, I'm ok with the other changes.
